### ISSUE TITLE
Erroneous Compiler Error

### ISSUE THREAD

**Issue by PrestonHamlin** [Microsoft/DirectXShaderCompiler#1512]
Typed loads with a template argument of uint64_t4 fail with an error message about HLSL version, even when HLSL 2018 is specified. It would seem that the type is not being recognized, but the error is misleading.

`error.hlsl:6:30: error: Explicit template arguments on intrinsic Load requires HLSL version 2018 or above.
  uint64_t4 val = g_bab.Load<uint64_t4>(addr);
`

```
RWByteAddressBuffer g_bab : register(u0);

[numthreads(8,8,1)]
void main(uint GI : SV_GroupIndex) {
  uint addr = GI * 8;
  uint64_t4 val = g_bab.Load<uint64_t4>(addr);
  g_bab.Store(addr, val.x + 1);
}
```

**Comment by PrestonHamlin** [comment]
Note that this occurs for me in the dxr-master branch, where there is some support for SM6.3 64-bit raw buffer load/store operations.

**Comment by MrTrillian** [comment]
The error does not repro on the latest master anymore, however I filed the following related bug: https://github.com/Microsoft/DirectXShaderCompiler/issues/1972
